### DISCUSSION TITLE
RSS feeds from Oxford Academic can't be added

### DISCUSSION THREAD

**Discussion by panyq357** [FreshRSS/FreshRSS#5060] [Q&A] [answered]
Hello, I was trying to add [this feed](https://academic.oup.com/rss/site_6317/4077.xml) from [The Plant Cell](https://academic.oup.com/plcell/issue/) (a academic journal), but failed. 

The log FreshRSS outputted looks like this:

```
Unknown error for feed [https://academic.oup.com/rss/site_6317/4077.xml]
```

After appending `#force_feed`, the log looks like this:

```
A feed could not be found at `https://academic.oup.com/rss/site_6317/4077.xml`. This does not appear to be a valid RSS or Atom feed. [https://academic.oup.com/rss/site_6317/4077.xml#force_feed]
```

But this feed did passed the check from W3C: <https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Facademic.oup.com%2Frss%2Fsite_6317%2F4077.xml>

So, maybe the RSS generator that Oxford Academic used is not compatible with FreshRSS's parser? 

I wonder if there's a workaround?

**Comment by Alkarex** [comment, marked as answer]
Hello,
Wait one hour, and then try again by putting a custom user agent string in the feed's advanced settings in FreshRSS, using the same user agent than the one of your Web browser.
It looks like the server is filtering clients based on user-agent string.

**Comment by panyq357** [comment]
I found the problem: Oxford Academic has banned almost all IPs of all VPS providers, and that's why my FreshRSS (deployed in a VPS) can't get the real feed contents.
